packages/zk-wizard/src/zk-wallet-account.ts

```diff
@@ -111,7 +113,13 @@ export class ZkWalletAccount {
   }
 
   addERC721(...addresses: string[]) {
-    this.erc721.push(...addresses.map(Address.from))
+    this.erc721.push(
```

Comment:
Added same logic to `addERC721` for consistency. There was not filter earlier.

Reply:
Yeah, I could not find a reason why without using the `filter`
It looks good.